<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainGameWindowClass</class>
 <widget class="QMainWindow" name="MainGameWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainGameWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/newPrefix/GameImages/gameLobby.jpg);
background-repeat: no-repeat;
background-position: center;
background-size: cover;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="findGame">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
border: 1px solid rgba(255, 255, 255, 80);
background: rgba(30, 30, 60, 120);
border-radius: 10px;
padding: 6px;
</string>
    </property>
    <property name="text">
     <string>Find - Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createGame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
border: 1px solid rgba(255, 255, 255, 80);
background: rgba(30, 30, 60, 120);
border-radius: 10px;
padding: 6px;
</string>
    </property>
    <property name="text">
     <string>Create - Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="account">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
border: 1px solid rgba(255, 255, 255, 80);
background: rgba(30, 30, 60, 120);
border-radius: 10px;
padding: 6px;
</string>
    </property>
    <property name="text">
     <string>Your account</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeDate">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>230</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">text-align: center;
color: white;
font-weight: bold;
border: 1px solid rgba(255, 255, 255, 80);
background: rgba(30, 30, 60, 120);
border-radius: 10px;
padding: 6px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="logoutButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>60</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tahoma</family>
      <pointsize>8</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Logout</string>
    </property>
   </widget>
   <widget class="QLabel" name="messageWelcome">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>491</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>22</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; 
font-weight: bold; 
border: 1px solid rgba(255, 255, 255, 80); 
background: rgba(30, 30, 60, 120); 
border-radius: 10px; 
padding: 6px;</string>
    </property>
    <property name="text">
     <string>You are now in the lobby! ðŸ‘€</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="randomGame">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Fluent Icons</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-weight: bold;
border: 1px solid rgba(255, 255, 255, 80);
background: rgba(30, 30, 60, 120);
border-radius: 10px;
padding: 6px;
</string>
    </property>
    <property name="text">
     <string>Random - Game</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
